specification/monitor/resource-manager/Microsoft.Insights/stable/2024-02-01/metricNamespaces_API.json

```diff
@@ -0,0 +1,177 @@
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/{resourceUri}/providers/microsoft.insights/metricNamespaces": {
+      "get": {
+        "tags": [
+          "metricNamespaces"
+        ],
+        "operationId": "MetricNamespaces_List",
+        "description": "Lists the metric namespaces for the resource.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceUriParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/StartTimeParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Successful request to get the list of metric namespaces",
+            "schema": {
+              "$ref": "#/definitions/MetricNamespaceCollection"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "x-ms-examples": {
+          "Get Metric Namespaces without filter": {
+            "$ref": "./examples/GetMetricNamespaces.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "MetricNamespaceName": {
+      "properties": {
+        "metricNamespaceName": {
+          "type": "string",
+          "description": "The metric namespace name."
+        }
+      },
+      "description": "The fully qualified metric namespace name."
+    },
+    "NamespaceClassification": {
+      "type": "string",
+      "description": "Kind of namespace",
+      "enum": [
+        "Platform",
+        "Custom",
+        "Qos"
+      ],
+      "x-ms-enum": {
+        "name": "NamespaceClassification",
+        "modelAsString": true
+      }
+    },
+    "MetricNamespace": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The ID of the metric namespace."
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of the namespace."
+        },
+        "name": {
+          "type": "string",
+          "description": "The escaped name of the namespace."
+        },
+        "classification": {
+          "$ref": "#/definitions/NamespaceClassification",
```

Comment:
classification is not a top level property in ARM envelopes. pl move it inside the properties bag

Reply:
or did u mean for this to be the "kind" property?

Reply:
I can't change this, that would be a breaking change. I realize the diff tool is broken but the only change between this and the 2023-10-01 version is the version. This is just a version rev with no other changes.